### PR TITLE
Add former nimi.li-exclusive words to sandbox

### PR DESCRIPTION
**Pull request by cubedhuang**
i am unsure about a few things
- what should i do with `[usage]` since it has never (to my knowledge) been recorded? (i just left it empty)
  - actually i just realized that these were probably surveyed in 2023 - do we still have those results?
- what do i do with no `ligatures`? should i remove it or leave it empty (i left them empty)
- do i need to add empty `sp_etymology` and `commentary` entries or can i just add nothing? (i created empty entries)

### CODE REVIEW

**Comment by TheOnlyTails** [comment]
All three of the things you listed were correct, just add an empty `audio` list to everything:
```toml
audio = []
```

File: words/metadata/aku.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2023"
+creator = ["kulupu Solali"]
+see_also = []
+source_language = "Romanian"
+usage_category = "sandbox"
+word = "aku"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/aku.toml:14]:
```suggestion
audio = []
```

File: words/metadata/anta.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2022"
+creator = ["ilo Powa"]
+see_also = []
+source_language = "Okinawan"
+usage_category = "sandbox"
+word = "anta"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/anta.toml:14]:
```suggestion
audio = []
```

File: words/metadata/inta.toml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2022"
+creator = ["ilo Powa"]
+see_also = ["kin", "taso"]
+source_language = "Portuguese"
+usage_category = "sandbox"
+word = "inta"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/inta.toml:14]:
```suggestion
audio = []
```

File: words/metadata/iseki.toml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2023"
+creator = ["kulupu Solali"]
+see_also = ["namako"]
+source_language = "Turkish"
+usage_category = "sandbox"
+word = "iseki"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/iseki.toml:14]:
```suggestion
audio = []
```

File: words/metadata/kana.toml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2022"
+creator = ["ilo Powa"]
+see_also = ["oni", "powe", "jume"]
+source_language = "Tulu"
+usage_category = "sandbox"
+word = "kana"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/kana.toml:14]:
```suggestion
audio = []
```

File: words/metadata/nowi.toml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2023"
+creator = ["ilo Powa", "kulupu Solali"]
+see_also = ["esun"]
+source_language = "Vietnamese"
+usage_category = "sandbox"
+word = "nowi"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/nowi.toml:14]:
```suggestion
audio = []
```

File: words/metadata/sole.toml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2022"
+creator = ["waso Netun"]
+see_also = []
+source_language = "Spanish"
+usage_category = "sandbox"
+word = "sole"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/sole.toml:14]:
```suggestion
audio = []
```

File: words/metadata/ta.toml [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+book = "none"
+coined_era = "post-ku"
+coined_year = "2022"
+creator = ["ilo Powa"]
+see_also = []
+source_language = "a priori"
+usage_category = "sandbox"
+word = "ta"
+deprecated = false
+
```
Comment (TheOnlyTails) [inline words/metadata/ta.toml:14]:
```suggestion
audio = []
```

**Comment by cubedhuang** [comment]
a wait the other metadata things currently have a blank `[[audio]]` rather than a `audio = []`

should the format be consistent or is this fine?

**Comment by TheOnlyTails** [comment]
If you were to do just a `[[audio]]`, it would be the equivalent of doing `audio = [{}]`, which isn't what we want.
We do it for the etymology field because we need to have the same number of items in both the localized and non-localized etymology fields.

File: words/translations/en/etymology.toml [hunk 9]
```diff
@@ -143,6 +151,10 @@ language = "German"
 definition = "inside, center, stomach"
 language = "Tok Pisin"
 
+[[insa]]
```
Comment (TheOnlyTails) [inline words/translations/en/etymology.toml:154]:
```suggestion
[[inta]]
```

File: words/source/sp_etymology.toml [hunk 10]
```diff
@@ -4,13 +4,15 @@ a = "exclamation radical + Latin lowercase letter A; radical from exclamation po
...
 ako = "exclamation radical + Hangul syllable 아 (a); radical from exclamation point (!), without the full stop at the bottom. compare a, o"
+aku = ""
 ala = "X mark, a crosslinguistic symbol for negation"
 alasa = "pictogram of a bow and an arrow"
 ale = "infinity symbol (∞)"
 alente = ""
 ali = "infinity symbol (∞)"
 alu = ""
 anpa = "location radical + \"container\"; the latter perhaps from a truncated square for a general object (compare sinpin) or a container (compare poki). for the radical, see also how dots are used in blissymbols; compare blissymbol \"below\""
+ante = ""
```
Comment (TheOnlyTails) [inline words/source/sp_etymology.toml:15]:
```suggestion
anta = ""
```

**Review by TheOnlyTails — CHANGES_REQUESTED** [review]

**Comment by TheOnlyTails** [comment]
did you forget aku somewhere?

**Comment by cubedhuang** [comment]
i thought it was mad since i removed all the words from the `translations` folder, so now they're only in the `source` folder

should i leave both the `translations` and `source` entries?

**Comment by TheOnlyTails** [comment]
no, i think we just need to figure out a way to have crowdin generate the translation files at build time

**Review by gregdan3 — APPROVED** [review]

**Pull request merged**